docs/modules/rootless_rootfull.md

```diff
@@ -5,31 +5,96 @@ nav_order: 2
 parent: Modules
 ---
 
-## Rootless vs Rootfull
+# Rootless vs Rootfull
+
+In the Podman world, non-privileged users can run containers on their own:
+those are the so-called *rootless* containers.  In NethServer 8, we borrow
+the same word from Podman and use it in the context of modules, together
+with its opposite, *rootfull*.
+
+To inspect and modify a rootless module start a SSH session. SSH is
+preferred to `su - <user>` because the latter does not properly initialize
+the Systemd session environment. For instance, to check if Traefik is running:
+
+    ssh traefik1@localhost
+    systemctl --user status
+
+Let's see the differences of rootless modules vs rootfull modules.
+
+## Unix user
 
 The main difference between *rootless* and *rootfull* modules, as
 suggested by the adjective, is the Unix user running the module processes
 and its system privileges.
 
-**Rootfull** modules run as `root` (EUID 0).
+Rootfull modules run as `root` (EUID 0).
 
-**Rootless** modules run as a normal Unix user. The Unix user account is
```

Comment:
I'd prefer with bold format.

Reply:
I also suggest using a dots list like this:
* **Rootfull**: modules run as `root` (EUID 0).
* **Rootless**: modules run as a normal Unix user. The Unix user account is